modules/web/src/app/cluster/details/cluster/edit-cluster/component.ts

```diff
@@ -363,6 +374,11 @@ export class EditClusterComponent implements OnInit, OnDestroy {
     }
   }
 
+  isAllowedIPRangeSupported(): boolean {
```

Comment:
This can also be just a field instead of method. It will provide minor performance benefit since we won't be searching through the array on every change detection.